### ISSUE TITLE
I want to ask you some questions

### ISSUE THREAD

**Issue by sabaizzz** [tsmotlp/DRGAN#1]
Question 1: Why is there a prompt of missing'trainN' when I run the program, but I did not see it in the paper
Question 2: Since it is unsupervised learning, why does x have a label? And what oct image should be put in the y folder?
thank you

**Comment by tsmotlp** [comment]
> Question 1: Why is there a prompt of missing'trainN' when I run the program, but I did not see it in the paper
> Question 2: Since it is unsupervised learning, why does x have a label? And what oct image should be put in the y folder?
> thank you

**Comment by yqhuang2912** [comment]
> Question 1: Why is there a prompt of missing'trainN' when I run the program, but I did not see it in the paper
> Question 2: Since it is unsupervised learning, why does x have a label? And what oct image should be put in the y folder?
> thank you

1. "trainN" is the pure noise patch mentioned in the paper
2. "x_label" is used to calculate the evaluation metrics for supervised methods in comparison. The y folder contains clean OCT images that do not match the content of the noisy OCT image x. 
please go through the paper again and the detailed descriptions of your questions are given in section "Data preparation".

**Comment by hamu12md** [comment]
I kind of get it, what you mean is that the label image is the GT image corresponding to the noise image, if not, please correct

**Comment by tsmotlp** [comment]
Yes, the label images refer to well-aligned clean images obtained by registering and averaging multiple samples taken from the same position of the same subject (which you referred to as GT images). Please note that these label images are only used to calculate the evaluation metrics for supervised methods in comparison and are not involved in the training process.

**Comment by yqhuang2912** [comment]
> I kind of get it, what you mean is that the label image is the GT image corresponding to the noise image, if not, please correct

To some extent, yes, due to the inability to obtain true label images in clinical settings, the label images refer to well-aligned clean images obtained by registering and averaging multiple samples taken from the same position of the same subject (which you referred to as GT images). Please note that these label images are only used to calculate the evaluation metrics for supervised methods in comparison and are not involved in the training process.

**Comment by hamu12md** [comment]
Thank you for your answer, the point I don't understand at the moment is that the noise-only patch in your code is cut from the background part of the noisy image? (I experimented, but the model didn't run out very well)
